code/__DEFINES/colors.dm

```diff
@@ -26,6 +26,8 @@
 #define COLOR_BLUE_LIGHT       "#33CCFF"
 #define COLOR_NAVY             "#000080"
 #define COLOR_PINK             "#FFC0CB"
+#define COLOR_BLUSH_PINK       "#DE5D83"
+#define COLOR_BLUSH_POLYSMORPH "#00FF32"
```

Comment:
put this on their own category this is way too far out of place

Reply:
how about moving it to the bottom of the color defines, since as far as i can tell, every single color define is in the defines/colors.dm? so i'm not keen on this being the only couple of color defines not found in the colors file.

Reply:
i can't make a suggestion for the bottom of the file since you only changed this part